src/components/language-selector.jsx

```diff
@@ -12,18 +12,27 @@ function LanguageSelector({ languageList, slug }) {
 
   const [isOpen, setIsOpen] = useState(false);
 
+  /* when a language button is clicked, it navigates to the url
+  with the language code of the clicked button and the slug of the current page */
   const handleClick = (langKey) => {
     i18n.changeLanguage(langKey, navigate(`/${langKey}/${slug}`));
   };
 
+  // create the language selector buttons based on the language list (languages used in src/locales)
+  // for 5 or less languages, the buttons are simply in the banner at the top of the page
+  // for 5 or more languages, the buttons are in a dropdown menu
   const languageMenuItems = languageList.map((language) => (
     <div
       className={
-        languageList.length <= 5
-          ? 'banner__language-selector-item'
-          : i18n.dir(language.langKey) === 'rtl'
-            ? 'banner__language-selector-item--rtl'
-            : 'banner__language-selector-item--ltr'
+        `${languageList.length <= 5 // are there are less than 5 languages used in the locales folder
```

Comment:
`// less than 5 languages used in the locales folder`

Reply:
changed to "5 or less"